dds_web/templates/technical_overview.html

```diff
@@ -0,0 +1,34 @@
+{% extends "base.html" %}
+
+{% block body %}
+<html>
+    <div class="container p-0" style="margin-top: 30px; font-size: medium;">
+        <h2>Technical overview ot the Data Delivery System</h2>
+        <p class="lead">To read the Technical overview of the DDS, please follow the link <a href="https://scilifelab.atlassian.net/wiki/spaces/deliveryportal/pages/2201944070/2022+June+-+Technical+Overview" target="_blank">Technical overviview</a>.&nbsp;</p>
+        <p class="lead">Alternatively, you can download a PDF version on the document usinng the link <a href="https://scilifelab.atlassian.net/wiki/download/attachments/2201944070/2022-June_Technical-Overview.pdf?api=v2">Technical overviview PDF version</a>.&nbsp;</p>
+    <br>
```

Comment:
```suggestion
        <h2>Technical Overview of the Data Delivery System</h2>
        <p>The technical overview provides an overview of how the DDS works and is meant to be used. The document can be found on Confluence or downloaded as a PDF.</p>
        <p class="lead"><a class="btn btn-success" href="https://scilifelab.atlassian.net/wiki/spaces/deliveryportal/pages/2201944070/2022+June+-+Technical+Overview" target="_blank">Read on Confluence</a>&nbsp;<a class="btn btn-success" href="https://scilifelab.atlassian.net/wiki/download/attachments/2201944070/2022-June_Technical-Overview.pdf?api=v2">Download the PDF</a>&nbsp;</p>
        <br>
```

Reply:
Except maybe change from `btn-success` to `btn-primary` or `btn-info`: https://www.w3schools.com/bootstrap/bootstrap_buttons.asp

Reply:
sure, I wasn't really sure I understand how you want it to look, so I decided to at least create the button and the page and put smth there

Reply:
Yeah it was good. Wasn't actually sure how I wanted it to look either tbh, just thought of it now when reviewing. This was just a slight simplification / makes it easier to quickly find the info imo. What do you think?

Reply:
Looks good with buttons, I decided to go for btn-info